.github/workflows/test.yml

```diff
@@ -57,17 +57,17 @@ jobs:
       image: rostooling/setup-ros-docker:ubuntu-focal-latest
       options: -u root  # setup-node requires root access
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
       with:
         path: '.'
-    - uses: actions/setup-node@v3.5.1
+    - uses: actions/setup-node@v4.0.3
       with:
-        node-version: '12.x'
+        node-version: '20.x'
```

Comment:
While we're not using Node.js v12 anymore, we're using v16, so we probably shouldn't install v20 here: https://github.com/ros-tooling/action-ros-lint/blob/382ad401d327e97060a978cb7d314c536f2d5ff0/package.json#L38

Reply:
For now I have reduced the patch version from 12 to 16.